lab-judy/server.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+//node modules
+
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+//npm modules
+const cowsay = require('cowsay');
+
+console.log(process.argv[0]);
+
+
+//app modules
+const parseBody = require('./lib/parse-body.js');
+
+//module constants
+const PORT = process.env.PORT || 3000;
+
+//module logic
+const server = http.createServer(function(req, res){
+
+  req.url = url.parse(req.url);
+  req.url.query = queryString.parse(req.url.query);
+
+  if(req.method === 'GET' && req.url.pathname === '/'){
```

Comment:
The lab directions specify that **any** request to '/' should respond this way, not just a GET request.